_posts/2019-09-19-moveit-workshop-macau.md

```diff
@@ -0,0 +1,32 @@
+---
+author: Rob Coleman
+comments: false
+date: 2019-09-19
+layout: post
+title: MoveIt Workshop 2019 Macau
+media_type: image
+media_link: /assets/images/blog_posts/moveit_macau_2019/moveit-macau-2019-logo.png
+description: First annual developer meeting, co-located with ROSCon
+categories:
+- MoveIt
+- ROS
+---
+
+The first annual MoveIt Workshop 2019 Macau will be on Saturday, November 2, 2019, the day after ROSCon. Sponsored by PickNik Consulting, it is an informal day of presentations, panel discussions, and hands-on training of latest MoveIt features from some of the top thinkers in the ROS community. Some topics to be covered include:
+
+* MoveIt 2 Progress and Direction	
+* Intel's Work on MoveIt and ROS2 Grasping
+* New Inverse Kinematics Plugin API for Constraint-Based Solvers and Velocity IK	
```

Comment:
I'm not going to present a ready-for-production IK API, but only some suggestions as a basis for further discussions.
```suggestion
* Discussing a new Inverse Kinematics plugin API to support constraint-based solvers and velocity IK
```